### ISSUE TITLE
[Suggestion] Add the feature of one-click installation of magisk and gapp

### ISSUE THREAD

**Issue by beilunyang** [alesimula/wsa_pacman#86]
I think we can add one click install magisk and gapp function to facilitate users to root android.
Refer to this project for details.  https://github.com/LSPosed/MagiskOnWSA

**Comment by alesimula** [comment]
@beilunyang that's a feature i was actually thinking of adding, but there are a few issues: in order to install Magisk or Play Store, you actually need to uninstall the official WSA app and install an unofficial build (the project you linked)

That project is already a one-click install

I tried a few other ways to install the Play Store on the official build but I was unsuccessful

**Comment by beilunyang** [comment]
That's true. But I don't care if wsa is official or not. I just want to easily install magisk and gapp.
That project is not very easy to use, you need to clone the project and compile it yourself, then unzip it and run the install script.
I think we can compile several versions in advance. Then integrate it into wsa_pacman
to make it easier for users to use.
And I think wsa_pacman can integrate more adb related functions, so that users can interact with wsa and don't need to know adb commands.
@alesimula

**Comment by alesimula** [comment]
@beilunyang whixh adb functions you think would ne useful to nave on WSA?

Also i'll think about Magisk + Gapps, bit at the moment it seems a bit complicated

**Comment by beilunyang** [comment]
Example: 
```
adb logcat
adb shell ps
adb push
adb pull 
```
Of course, I also think it's possible to provide a way for users to add adb commands to wsa themselves. That way, instead of typing similar adb commands over and over again each time, the user would just click the button they configured in wsa
@alesimula
